pkg/upgradeconfigmanager/upgradeconfigmanager.go

```diff
@@ -30,7 +30,7 @@ var log = logf.Log.WithName("upgrade-config-manager")
 
 const (
 	// Name of the Custom Resource that the provider will manage
-	UPGRADECONFIG_CR_NAME = "osd-upgrade-config"
+	UPGRADECONFIG_CR_NAME = "managed-upgrade-config"
```

Comment:
I feel like this should be defined at higher level:https://github.com/openshift/managed-upgrade-operator/blob/master/config/config.go ?

Reply:
+1